gapis/resolve/dependencygraph2/memory.go

```diff
@@ -197,6 +202,8 @@ func observationSlice(obs api.CmdObservation) memory.Slice {
 	return memory.NewSlice(obs.Range.Base, obs.Range.Base, obs.Range.Size, obs.Range.Size, obs.Pool, reflect.TypeOf(memory.Char(0)))
 }
 
+// applyMemWrite marks or creates the memoryWrites of wmap affected by (p, s)
```

Comment:
wmap is writeMap? Feels worth documenting

Reply:
done